Explorer/Assets/Scripts/CrdtEcsBridge/ComponentWriter/ECSToCRDTWriter.cs

```diff
@@ -0,0 +1,70 @@
+using CRDT;
+using CRDT.Protocol;
+using CRDT.Protocol.Factory;
+using CrdtEcsBridge.OutgoingMessages;
+using CrdtEcsBridge.Serialization;
+using System;
+using System.Collections.Generic;
+
+namespace CrdtEcsBridge.ECSToCRDTWriter
+{
+    public class ECSToCRDTWriter : IECSToCRDTWriter
+    {
+        private readonly ICRDTProtocol crdtProtocol;
+        private readonly IOutgoingCRTDMessagesProvider outgoingCRDTMessageProvider;
+        private readonly Dictionary<int, IComponentSerializer> serializers;
+
+        public ECSToCRDTWriter(ICRDTProtocol crdtProtocol, IOutgoingCRTDMessagesProvider outgoingCRDTMessageProvider)
+        {
+            this.crdtProtocol = crdtProtocol;
+            this.outgoingCRDTMessageProvider = outgoingCRDTMessageProvider;
+            serializers = new Dictionary<int, IComponentSerializer>();
+        }
+
+        public void PutMessage<T>(CRDTEntity crdtID, int componentId, T model)
+        {
+            if (serializers.TryGetValue(componentId, out IComponentSerializer serializer))
+                ProcessMessage(crdtProtocol.CreatePutMessage(crdtID, componentId, serializer.Serialize(model)));
+            else
+                throw new Exception($"Serializer not present for type {typeof(T).Name}");
+        }
+
+        public void AppendMessage<T>(CRDTEntity crdtID, int componentId, T model)
+        {
+            if (serializers.TryGetValue(componentId, out IComponentSerializer serializer))
+                ProcessMessage(crdtProtocol.CreateAppendMessage(crdtID, componentId, serializer.Serialize(model)));
+            else
+                throw new Exception($"Serializer not present for type {typeof(T).Name}");
+        }
+
+        public void DeleteMessage(CRDTEntity crdtID, int componentId)
+        {
+            ProcessMessage(crdtProtocol.CreateDeleteMessage(crdtID, componentId));
+        }
+
+        public void RegisterSerializer(int componentId, IComponentSerializer serializer)
```

Comment:
You don't need it: you can use `ISDKComponentsRegistry`, `SDKComponentBridge` already contains information about Serializer